corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -230,10 +233,18 @@ hqDefine("geospatial/js/geospatial_map", [
             });
 
             if ($data.length && map) {
-                var caseData = $data.data("context");
+                var contextData = $data.data("context");
                 map.clearMap();
-                cases = caseData.cases
+                cases = contextData.cases
                 map.addCaseMarkersToMap();
+
+                if (contextData.invalid_geo_cases_report_link) {
+                    var missingCasesLink = contextData.invalid_geo_cases_report_link;
+                    var missingCasesLinkTag = "<a href=" + missingCasesLink + ">" + gettext("View here") + "</a>"
```

Comment:
Nit: You could replace this to use JavaScript string formatting. This can then be changed to something like: ``var missingCaseLinkTag = `<a href="${missingCasesLink}">${gettext("View here")}</a>`; ``